.github/workflows/checkov.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: "recursive"
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.ENGINEERING_GITHUB_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Why Checkov needs personal token?

Reply:
No need, didn't realise this change reverted it, thanks for pointing out .